ddtrace/contrib/kafka/patch.py

```diff
@@ -167,12 +183,16 @@ def traced_poll(func, instance, args, kwargs):
     if not pin or not pin.enabled():
         return func(*args, **kwargs)
 
-    with pin.tracer.trace(
-        schematize_messaging_operation(kafkax.CONSUME, provider="kafka", direction=SpanDirection.PROCESSING),
+    message = func(*args, **kwargs)
```

Comment:
Do we want to move the execution of the poll method (`func(*args, **kwargs)`) outside of the span? 

If we do then the duration of the span will not be accurate

Reply:
We are recalculating the start time and manually setting this attribute on the span to make it accurate